Downward mono cam + aruco tag

dakejahl opened the pull request:
Useful for testing precision landing in simulation

Jaeyoung-Lim commented on worlds/default.sdf:
```diff
@@ -238,5 +238,9 @@
+
+    <include>
+      <uri>model://arucotag</uri>
+    </include>
```
Please create a separate world for the aruco tag. IMHO, the default world should be a minimalistic world where anybody can run. Specific worlds should be used for specific feature testing.

Jaeyoung-Lim reviewed (changes requested):
Thanks for the PR.

Is this supposed to replace https://github.com/PX4/PX4-gazebo-models/pull/40 ?

dakejahl commented:
@Jaeyoung-Lim thanks for the review. Yes it replaces #40. I removed the commented out code, adjusted camera Z position, and added an **aruco** world

Once this is merged I'll open a PR in PX4 to add the airframe file and add to the cmake so we can use
```
make px4_sitl_default gz_x500_mono_cam_down_aruco
```
